src/BlueSCSI.cpp

```diff
@@ -484,66 +541,88 @@ void findDriveImages(FsFile root) {
     // Valid file, open for reading/writing.
     file = new FsFile(SD.open(name, O_RDWR));
```

Comment:
Some archives have optical image files with the read only attribute checked. We'll need to support opening RO files in this initial setup.